src/constant/index.js

```diff
@@ -540,6 +540,45 @@ const ERRORS = [
     description:
       "Displayed on the attempt of logging in. All seats within a given license are already taken.",
   },
+  {
+    id: "agent_offline",
+    message: "Agent offline",
+    description:
+      "Request couldn't be performed when agent is offline.",
+  },
+  {
+    id: "not_allowed",
+    message: "Not allowed",
+    description:
+      "Action is not allowed in current state.",
```

Comment:
Not sure if it's worth adding "in current state" if we don't give any details what that "current state" is

Reply:
I'd assume the state of the request 🤔
`The action is not allowed.`